libraries/ESP8266httpUpdate/examples/httpUpdateSecure/httpUpdateSecure.ino

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
While I'm happy to see someone using CertStore, for ESP8266 updates that's probably not what we want people to do since it's saying you'd accept an update from anyone on the internet.

What most people would want to do is have a single, private certificate that they generate and keep secure on their own company's system.  It would make this new example shorter and simpler, too, and focus it on the new way of doing the update vs. the overhead for making a CertStore...

Reply:
I do not understand:
You will not accept an update from anyone on the internet because the URL determines where you fetch the update, and the Client identifies the host. Also, there is no need to populate the cert store with all root certificates, just your company's one is possible too.
Do you want an extra, simpler example just using a cert? Could you provide it?